src/app/games/teeko/tests/TeekoMinimax.spec.ts

```diff
@@ -62,7 +62,7 @@ describe('TeekoMinimax', () => {
             // When calculating the board value
             const boardValue: number = minimax.getBoardValue(node).value;
 
-            // Then it should be the number of possible victory of 0 in negative
+            // Then it should be minus the number of possible victory of Player.ZERO in negative
```

Comment:
double négatif :D 
Then it should be the negative number of possible victory for Player.ZERO